packages/react/src/components/ComboBox/ComboBox-test.js

```diff
@@ -446,4 +446,140 @@ describe('ComboBox', () => {
       );
     });
   });
+
+  describe('ComboBox autocomplete', () => {
+    const items = [
+      { id: 'option-1', text: 'Option 1' },
+      { id: 'option-2', text: 'Option 2' },
+      { id: 'option-3', text: 'Option 3' },
+      { id: 'apple', text: 'Apple' },
+      { id: 'banana', text: 'Banana' },
+      { id: 'orange', text: 'Orange' },
+      { id: 'orangeish', text: 'Orangeish' },
+    ];
+
+    const mockProps = {
+      id: 'test-combobox',
+      items,
+      itemToString: (item) => (item ? item.text : ''),
+      onChange: jest.fn(),
+    };
+
+    it('should respect autocomplete prop', async () => {
+      render(<ComboBox {...mockProps} autocomplete />);
+      await waitForPosition();
+      const inputNode = findInputNode();
+      expect(inputNode).toHaveAttribute('autocomplete');
+    });
+    it('should use autocompleteCustomFilter when autocomplete prop is true', async () => {
+      render(<ComboBox {...mockProps} autocomplete />);
```

Comment:
and I think technically we're supposed to call setup before render now so that userEvent methods will all use the same user "instance"

```suggestion
      const user = userEvent.setup();
      render(<ComboBox {...mockProps} autocomplete />);
```

and then the usages below change too
```diff
- await userEvent.click(...
+ await user.click(...
```

Reply:
https://testing-library.com/docs/user-event/setup#starting-a-session-per-setup